sonar-xml-plugin/src/test/java/org/sonar/plugins/xml/XmlSensorTest.java

```diff
@@ -260,4 +276,22 @@ private void assertNoLog(String notExpected, boolean isRegexp) {
     }
   }
 
+  private void createXmlFile(int numberOfTags, String fileName) throws Exception {
+    init(false, temporaryFolder.getRoot());
+    File file = temporaryFolder.newFile(fileName);
+    String simpleTag = "<tag1 attr=\"val1\">text</tag1>\n";
+    String xml = "<?xml version=\"1.0\"?><root>\n";
+    StringBuilder str = new StringBuilder(xml);
```

Comment:
I would inline variables `xml` and `simpleTag`, both are used only once now, and it will make it simpler to read.